api/coreservice.go

```diff
@@ -1531,8 +1531,9 @@ func (core *coreService) ReadContractStorage(ctx context.Context, addr address.A
 
 // TODO: use EstimateGasForNonExecution in grpcserver.go
 // EstimateGasForNonExecution estimates action gas except execution
-func (core *coreService) EstimateGasForNonExecution(actType action.Action, payloadSize uint64) (uint64, error) {
+func (core *coreService) EstimateGasForNonExecution(actType action.Action, payload []byte) (uint64, error) {
```

Comment:
why change input type to `[]byte`? passing a `uint64` is more efficient than passing `[]byte`